### PR TITLE
Genetics Modification has a chance to turn people into corgis

### PR DESCRIPTION
**Pull request by chesse20**
[Changelogs]:

:cl: 
balance: 1% chance to turn occupants of genetic modifier into a corgi when modifying genes
/:cl:

[why]: Nerfs using dead bodies to spam genetics changes, buffs genetics traitors by allowing them to get rid of someone from being human/cloned by making them into a corgi.

### CODE REVIEW

File: code/game/machinery/computer/dna_console.dm [hunk 1]
```diff
@@ -562,6 +562,14 @@
 					viable_occupant.dna.unique_enzymes = buffer_slot["UE"]
 					viable_occupant.dna.blood_type = buffer_slot["blood_type"]
 
+	if(prob(1))
+		var/mob/living/carbon/human/H = viable_occupant
+		if(istype(H))
+			src.visible_message("<span class='warning'>The machine starts to malfunction!</span>")
```
Comment (deathride58) [inline code/game/machinery/computer/dna_console.dm]: You don't need src here

**Comment by Grazyn** [comment]
This seems like an all-around nerf to genetics though, not just to "using dead bodies"? And is there even a way to turn back into a human from corgi? Or is it just "whenever you use genetics on yourself or others, there is a 1% chance that your/their round is over"

Not a great balance change imho

File: code/game/machinery/computer/dna_console.dm [hunk 2]
```diff
@@ -562,6 +562,14 @@
 					viable_occupant.dna.unique_enzymes = buffer_slot["UE"]
 					viable_occupant.dna.blood_type = buffer_slot["blood_type"]
 
+	if(prob(1))
```
Comment (deathride58) [inline code/game/machinery/computer/dna_console.dm]: This chance is extremely high for something that's supposed to stop spam. If you've ever played geneticist, you'd know how often you have to reapply buffers to yourself if you want powers. Corgization is pretty much irreversible without dying, so this is a pretty major nerf to normal genetics gameplay.

Comment (chesse20) [reply code/game/machinery/computer/dna_console.dm]: Yeap

**Comment by lordsaladin** [comment]
We need more corgi prs.

**Comment by Iamgoofball** [comment]
not only is this furry crap, this makes genetics a pain

**Comment by Spaceinaba** [comment]
genetics has always been furry crap goof you can turn into monkeys you dork

**Comment by Iamgoofball** [comment]
that's classic sci-fi shit though, you know damn well it is

**Comment by YakumoChen** [comment]
Pointless change, you can already turn someone into a monkey and butcher the corpse if you want them to stay dead, and we ALSO already have gorillification for the same sort of purpose to prevent genetics spam.

**Comment by Grazyn** [comment]
@Repukan imagine if the KA had a 1% chance to turn you into a corgi every time you use it. You'd think it was unfair. Well, such a change would still be less impactful than this PR, because you could still use a resonator, a crusher,  a cutter or any other mining tool. Geneticists only have the modifier.

Not only that, but the reason provided isn't enough to warrant such a change. If pulsing dead bodies is a problem (I didn't even know it was a thing) just make the modifier not work on corpses. If genetics is so OP, then nerf the mutations. This is just a random FUCK YOU to a job that is already an RNG chore, under the guise of a balance change.

**Comment by PKPenguin321** [comment]
This is horrible gameplay-wise

**Comment by Shadowflame909** [comment]
on the note of gorillas. (DID you see the new song?) Has anyone ever been turned into one? It's pretty much impossible thanks to that rad nerf awhile back. So you don't see naturally spawned gorillas anymore...

**Comment by Ratvar** [comment]
Maybe reset server with 10% chance instead?

**Comment by Tlaltecuhtli** [comment]
:eggplant:

**Comment by optimumtact** [comment]
go back to citadel

**Pull request closed without merging**